src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -34,6 +34,41 @@
         </v-card-text>
       </template>
 
+      <!-- Mobile Tables -->
```

Comment:
I see almost no difference between mobile and desktop tables... just that the mobile v-simple-table is nested in a row+col <<< won't this work on desktop as well?

Reply:
Yea, the only difference is the grid system. 
I couldn't get it to preserve it's original layout on Desktop no, it stacks them as it would mobile. 
I can look into it some more but i didn't want to rebuild the table and how/where it loops. I will look again though.

Reply:
It might fit if we loop the columns instead of the table. Will try

Reply:
Updated this to the above. It works pretty much the same. When it breaks into two rows on an ipad, the upper and lower columns don't line up perfectly because there is different column widths but it's definitely good enough.